### PR TITLE
arguments

### PR DESCRIPTION
**Pull request by bukzor**

### CODE REVIEW

**Comment by kentwills** [comment]
@bukzor shipit

File: pgctl/cli.py [hunk 1]
```diff
@@ -4,114 +4,78 @@
...
 
 
 def main(argv=None):
-    args = parser().parse_args(argv)
-    config = PGCTL_CONFIG.combined()
+    p = parser()
+    args = p.parse_args(argv)
+    config = Config('pgctl')
+
+    print(config.from_app())
```
Comment (kentwills) [inline pgctl/cli.py]: debug comment, I will delete in my branch if you merge this in.

**Comment by bukzor** [comment]
Apparently coveralls is down for the day. http://status.coveralls.io/

The second commit makes coverage look better:

```
Name                                                                                       Stmts   Miss Branch BrMiss  Cover   Missing
--------------------------------------------------------------------------------------------------------------------------------------
/home/travis/build/Yelp/pgctl/.tox/python/lib/python2.7/site-packages/pgctl/__init__           0      0      0      0   100%   
/home/travis/build/Yelp/pgctl/.tox/python/lib/python2.7/site-packages/pgctl/cli               49      0      2      0   100%   
/home/travis/build/Yelp/pgctl/.tox/python/lib/python2.7/site-packages/pgctl/config            97      3     34      0    98%   21-23
/home/travis/build/Yelp/pgctl/.tox/python/lib/python2.7/site-packages/pgctl/configsearch      54      0     27      0   100%   
/home/travis/build/Yelp/pgctl/tests/integration/__init__                                       0      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/integration/cli                                           12      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/integration/config                                        35      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/integration/configsearch                                  19      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/integration_test                                           7      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/main_test                                                 40      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/testing/install_coverage_pth                               8      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/unit/__init__                                              0      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/unit/config                                               99      0      0      0   100%   
/home/travis/build/Yelp/pgctl/tests/unit/configsearch                                         43      0      4      0   100%   
--------------------------------------------------------------------------------------------------------------------------------------
TOTAL                                                                                        463      3     67      0    99%   
```

**Comment by kentwills** [comment]
nice, shipit.

**Pull request merged**